yarn.lock

```diff
@@ -9805,9 +9805,9 @@
   integrity sha512-+0autS93xyXizIYiyL02FCY8N+KkKPhILhcUSA276HxzreZ16kl+cmwvV2qAM/PuCCwPXzOXOWhiPcw20uSFcA==
 
 "@types/react-dom@17.0.21", "@types/react-dom@<18.0.0", "@types/react-dom@^17.0.21":
-  version "17.0.21"
-  resolved "https://registry.yarnpkg.com/@types/react-dom/-/react-dom-17.0.21.tgz#85d56965483ce4850f5f03f9234e54a1f47786e5"
-  integrity sha512-3rQEFUNUUz2MYiRwJJj6UekcW7rFLOtmK7ajQP7qJpjNdggInl3I/xM4I3Hq1yYPdCGVMgax1gZsB7BBTtayXg==
+  version "17.0.22"
```

Comment:
I suggest not to merge `yarn.lock` changes now, they are source of a lot of merge conflicts and runtime issues in Backstage in general. 
It would be safer/easier to rebuild the `yarn.lock` at once at the end, when the PR is really ready.

Reply:
This is dangerous because the CI we have here will fail.

Reply:
ok I will keep the change from my local build